zeppelin-web/src/app/notebook/paragraph/paragraph.css

```diff
@@ -396,6 +396,11 @@ table.table-shortcut {
  opacity: 0 !important;
 }
 
+#main .emacs-mode .ace_cursor {
+  background: #C0C0C0!important;
```

Comment:
It seems javascript sets the CSS in emacs mode.
![screen shot 2017-07-17 at 7 36 19 pm](https://user-images.githubusercontent.com/1881135/28298356-ad31b9a0-6b27-11e7-9a90-6f817ca63795.png)


If we want to retain the emacs mode and want the thin cursor at the same time, overriding emacs mode CSS would be a better approach.